draft-sharabayko-mops-srt.md

```diff
@@ -183,7 +183,8 @@ O (1 bit):
 KK (2 bits):
 : Encryption Flag. The flag bits indicate whether or not data is encrypted.
   The value "00b" means data is not encrypted, "01b" indicates that data is
-  encrypted with even key, and "11b" is used for odd key encryption.
+  encrypted with even key, and "10b" is used for odd key encryption.
+  The value "11b" is only used in control packets.
```

Comment:
encrypted with an even key, and "10b" is used for odd key encryption.
  The value "11b" (even and odd keys) is only used in KM control packets.